chromium_src/chrome/common/url_constants.cc

```diff
@@ -38,6 +40,9 @@ const char kChooserUsbOverviewURL[] =
 
 const char kChromeBetaForumURL[] =
     "https://community.brave.com/c/beta-builds";
+// TODO: replace w/link after marketing provides it.
+const char kChromeReleaseNotesURL[] =
+    "https://support.brave.com/";
```

Comment:
I think we have a link for this now... let me check

Reply:
https://brave.com/latest/

Reply:
Fixed by e3d0fa37c0cf5fa6ddde4e3a72040313e255ccca